patches/tools-metrics-histograms-enums.xml.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tools/metrics/histograms/enums.xml b/tools/metrics/histograms/enums.xml
+index e6ecb2ff8ef9ab8152cf50e51ac3de7ceb47e796..60001c81da986aba0e11f3c4d46108034f450128 100644
+--- a/tools/metrics/histograms/enums.xml
++++ b/tools/metrics/histograms/enums.xml
+@@ -35075,6 +35075,7 @@ from previous Chrome versions.
+   <int value="2141463681" label="enable-offer-upload-credit-cards"/>
+   <int value="2142661816" label="AllowRemoteContextForNotifications:disabled"/>
+   <int value="2142979536" label="EnableManualFallbacksFilling:disabled"/>
++  <int value="2143102221" label="enable-speedreader"/>
```

Comment:
I don't think this is needed, if we get rid of the feature flag at least.

Reply:
correct, this was a leftover from slightly earlier and simpler wire-up, now removed in favour content-settings based approach